beacon-chain/sync/initial-sync/service.go

```diff
@@ -268,24 +274,6 @@ func (s *InitialSync) checkInMemoryBlocks() {
 	}
 }
 
-// listenForNewBlocks listens for block announcements beyond the canonical head slot that may
```

Comment:
No longer relevant as this is handled by regular sync

Reply:
we still should have it, otherwise the node will always have to play catch up whenever it starts regular sync